package.json

```diff
@@ -34,6 +34,8 @@
   },
   "dependencies": {
     "classnames": "^2.2.6",
+    "date-fns": "^2.17.0",
+    "date-fns-tz": "^1.1.1",
```

Comment:
This is to get the formatting that we want, and also to move away from `moment`, which is in [maintenance mode only](https://momentjs.com/docs/#/-project-status/).